atomix/cluster/src/main/java/io/atomix/cluster/protocol/SwimMembershipProtocol.java

```diff
@@ -611,7 +632,7 @@ private void failProbes(final ImmutableMember suspect) {
    */
   private CompletableFuture<Boolean> requestProbe(
       final SwimMember member, final ImmutableMember suspect) {
-    LOGGER.debug("{} - Requesting probe of {} from {}", localMember.id(), suspect, member);
+    PROBE_LOGGER.info("{} - Requesting probe of {} from {}", localMember.id(), suspect, member);
```

Comment:
:wrench: As we always probe from multiple members, I would rather combine the state into a single one at the source, i.e. `requestProbes`. Something like: " Requesting probe of Member{id=1, address=dd-swim-logs-zeebe-1.dd-swim-logs-zeebe.dd-swim-logs.svc:26502, properties={brokerInfo=EADJAAAAAwABAAAAAwAAAAMAAAADAAAAAAABCgAAAGNvbW1hbmRBcGk+AAAAZGQtc3dpbS1sb2dzLXplZWJlLTEuZGQtc3dpbS1sb2dzLXplZWJlLmRkLXN3aW0tbG9ncy5zdmM6MjY1MDEFAAMBAAAAAAIAAAAAAwAAAAEMAAIBAAAABAAAAAAAAAACAAAAAwAAAAAAAAAOAAAAOC4zLjAtU05BUFNIT1QFAAMBAAAAAQIAAAABAwAAAAE=}, version=8.3.0-SNAPSHOT, timestamp=1686042862870, state=ALIVE, incarnationNumber=1686042862876} from members Member{id=2, address=dd-swim-logs-zeebe-2.dd-swim-logs-zeebe.dd-swim-logs.svc:26502} and Member{id=3, address=dd-swim-logs-zeebe-3.dd-swim-logs-zeebe.dd-swim-logs.svc:26502}".

Reply:
:question: There's also no state change here, so I'm unsure why this is info and not debug :thinking: